src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/ApproveVacancyReviewCommandHandler.cs

```diff
@@ -88,7 +88,7 @@ private ClosureReason GetClosureReasonBasedOnTransferInfo(TransferInfo transferI
             }
             else
             {
-                return ClosureReason.BlockedByQa;
+                return ClosureReason.TransferredByQa;
```

Comment:
This could still be the result of BlockedByQa. Think this method needs to be rewritten and not exclusively depend on TransferReason alone to work out what the closure reason is. The particular case in this is that there is an open `UnderReview` vacancy review for an employer created vacancy for a provider which has been blocked by QA. The employer owned vacancy wouldn't have been transferred, just closed in this case.

Reply:
👍 Good spot